extensions/cdc-postgres/src/test/java/com/hazelcast/jet/cdc/postgres/AbstractPostgresCdcIntegrationTest.java

```diff
@@ -60,6 +61,12 @@ public abstract class AbstractPostgresCdcIntegrationTest extends AbstractCdcInte
                     .withStartupTimeoutSeconds(300)
     );
 
+    @BeforeClass
+    public static void ignoreOnArm64() {
+        //There is no working arm64 version of example-postgres image
```

Comment:
Can you please create
 an issue to start using official postgres images in the CDC tests.

Reply:
Created in Jira:
https://hazelcast.atlassian.net/browse/HZ-2060
I can adjust that in the future.